en-US/W.xml

```diff
@@ -109,23 +112,23 @@
 			<term>while</term>
 			 <listitem>
 				<para>
-		  In technical content, use “while” only to indicate that many tasks or processes occur at the same time.
-          For example, instead of "While most targets use one target portal group (TPG), advanced configurations might define multiple TPGs", write "Although most targets use one target portal group (TPG), advanced configurations might define multiple TPGs".
+		      In technical content, use "while" only to indicate that many tasks or processes occur at the same time.
+          For example, instead of "While most targets use one target portal group (TPG), advanced configurations might define multiple TPGs", write "Although most targets use one target portal group (TPG), advanced configurations might define multiple TPGs."
```

Comment:
I thought that was where they belonged. Did I read it wrong?

Reply:
The last example in this sentence could go either way, because it is a complete sentence in itself; however, my overriding preference is to move the punctuation outside the quotation marks, for consistency with the previous example that ends with a comma. A minor point for sure.